docs/cs-manual/source/tables.rst

```diff
@@ -0,0 +1,177 @@
+Tables
+######
+
+If you have launched OpenMW CS already and played around with it for a bit, you
+have probably gotten the impression that it contains lots of tables. You’d be
+spot on: OpenMW CS is built around using tables. This does not mean it works
+just like a spreadsheet application, though. It would be more accurate to think
+of databases instead. Due to the vast amounts of information involved with
+Morrowind tables just made the most sense. You have to be able to spot
+information quickly and be able to change them on the fly. Let’s browse through
+the various screens and see what all these tables show.
+
+
+Used Terms
+**********
+
+Record
+   An entry in OpenMW CS representing an item, location, sound, NPC or anything
+   else.
+
+Reference, Referenceable
+   When an item is placed in the world, it does not create a new record each
+   time. For example, the game world might contain a lot of exquisite belts on
```

Comment:
Technically it does create a new sub-record of type instance, which is actually handled just like a regular record in OpenMW-CS. We don't necessarily have to confuse the user with the details, but a less misleading wording would be helpful.